### ISSUE TITLE
Parsing of dependencyManagement doesnt work

### ISSUE THREAD

**Issue by lucastheisen** [lucastheisen/perl-maven#6]
It appears that `dependencyManagement` actually gets _munged_ with `dependencies` resulting in a really wacky outcome.

**Comment by lucastheisen** [comment]
Resolved by #7

**Comment by lucastheisen** [comment]
```
[Phase_Begins] ########## Release ##########
[@Filter/UploadToCPAN] registering upload with PAUSE web server
[@Filter/UploadToCPAN] POSTing upload for Maven-Agent-1.15.tar.gz to https://pause.perl.org/pause/authenquery
[@Filter/UploadToCPAN] PAUSE add message sent ok [200]
[Phase_Ends] ########## Release ##########
```
